Library/Formula/fish.rb

```diff
@@ -36,9 +36,16 @@ def caveats; <<-EOS.undent
       chsh -s #{HOMEBREW_PREFIX}/bin/fish
     to make fish your default shell.
 
-    If you are upgrading from an older version of fish, you should now run:
+    And optionally:
+      fish_config
+    to review configuration.
+
+    If you are upgrading from an older version of fish, you should:
+
+    1. Terminate the outdated fish daemon:
       killall fishd
```

Comment:
@mikemcquaid
Respectfully, I have ask "why?". We [nuked `pkill fishd`](https://github.com/Homebrew/homebrew/commit/8a108bed154ae79a766d59949f0e753484525a76#diff-f45e0a4db63db6d7ac3617f42ccf6192L32) with the 2.2.0 release; there's really no harm in having a zombie `fishd` (a few hundred K of memory, max) until the next time the user reboots; and since the daemon has been eliminated, it won't reoccur after that.

Again, no offense intended, I fail to see any good reason for merging this PR; it just adds 'caveats'  bloat, IMHO, as none of the steps outlined are "required" by any means.